neps/nep-0245.md

```diff
@@ -62,12 +62,12 @@ To recap, we choose to create this standard, to improve interoperability, develo
 ### MT Interface
 
 ```ts
-// The base structure that will be returned for a token. If contract is using
-// extensions such as Approval Management, Enumeration, Metadata, or other
-// attributes may be included in this structure.
+// The base structure that will be returned for a token.
 type Token = {
   token_id: string,
-  owner_id: string | null
+  owner_id: string | null, // not null if the token is a NFT
+  metadata: MTTokenMetadataAll | null // not null if Metadata extension is enabled
```

Comment:
How/when does this field get assigned to a token?

Reply:
Curious if we need to deal with Metadata at all at this stage, wouldn't it be nicer to go with iterative approach and get the barebone tokens right up to the contract implementation and then deal with extensions?